views/partials/subpages/unsubscribe_survey.hbs

```diff
@@ -0,0 +1,20 @@
+<section id="unsubscribe-survey" class="unsubscribe-survey section-wrapper drop-shadow">
+  <div class="whole">
+    <form action="" method="post" id="unsubscribe-survey-form" class="form-group">
+      <span class="secondary-title" tabindex="1">{{{fluentFormat req.supportedLocales "unsub-survey-form-label"}}}</span>
+      {{#each UNSUB_REASONS}}
+        <div class="radio-button-group" tabindex="1">
+          <input id="reason{{ id_number }}" type="radio" name="unsubscribe-reason" data-analytics-label="{{ value }}" value="{{getStringID ../req.supportedLocales "unsub-reason-" id_number}}" />
+          <span class="radio-dot"></span>
+          <label for="reason{{ id_number }}">{{{getStringID ../req.supportedLocales "unsub-reason-" id_number}}}</label>
```

Comment:
Regression: this re-introduced the bug where the user has to click on the `<label>` element - i.e., clicking on the `<span class="radio-dot">` element doesn't work anymore? Firefox Nightly 65.0a1 (2018-11-24)